text/0117-unbrick-collective.md

```diff
@@ -0,0 +1,184 @@
+
+## Summary
+
+A followup of the [RFC-0014]. This RFC proposes adding a new collective to the Polkadot Collectives
+Chain: The Unbrick Collective, as well as improvements in the mechanisms that will allow teams
+operating paras that had stopped producing blocks to be assisted, in order to restore the production
+of blocks of these paras.
+
+## Motivation
+
+Since the initial launch of Polkadot parachains, there has been many incidients causing parachains
+to stop producing new blocks (therefore, being _bricked_) and many occurrences that requires
+Polkadot governance to update the parachain head state/wasm. This can be due to many reasons range
+from incorrectly registering the initial head state, inability to use sudo key, bad runtime
+migration, bad weight configuration, and bugs in the development of the Polkadot SDK.
+
+Currently, when the para is not unlocked in the _paras registrar_[^1], the `Root` origin is required to
+perform such actions, involving the governance process to invoke this origin, which can be very
+resource expensive for the teams. The long voting and enactment times also could result significant
+damage to the parachain and users.
+
+Finally, other instances of governance that might enact a call using the `Root` origin (like the
+Polkadot Fellowship), due to the nature of their mission, are not fit to carry these kind of tasks.
+
+In consequence, the idea of a Unbrick Collective that can provide assistance to para teams when
+they brick and further protection against future halts is reasonable enough.
+
+## Stakeholders
+
+- Parachain teams
+- Parachain users
+- OpenGov users
+- Polkadot Fellowship
+
+## Explanation
+
+### The Collective
+
+The Unbrick Collective is defined as an unranked collective of members, not paid by the Polkadot
+Treasury. Its main goal is to serve as a point of contact and assistance for enacting the actions
+needed to unbrick a para. Such actions are:
+
+- Updating the Parachain Verification Function (a.k.a. a new WASM) of a para.
+- Updating the head state of a para.
+- A combination of the above.
+
+In order to ensure these changes are safe enough for the network, actions enacted by the Unbrick
+Collective must be whitelisted via similar mechanisms followed by collectives like the Polkadot
+Fellowship. This will prevent unintended, not overseen changes on other paras to occur.
+
+Also, teams might opt-in to delegate handling their para in the registry to the Collective. This
+allows to perform similar actions using the _paras registrar_, allowing for a shorter path to unbrick a
+para.
+
+Initially, the unbrick collective has powers similar to a parachains own sudo, but permits more
+decentralized control. In the future, Polkadot shall provide functionality like SPREE or JAM that
+exceeds sudo permissions, so the unbrick collective cannot modify those state roots or code.  
+
+### The Unbrick Process
```

Comment:
IMO this should still require that the chain is actually bricked, aka not producing blocks. When the chain enables the unbrick collective it could for example pass a time out after which the chain is seen as "bricked".